bytes/bytes/rabbitmq.py

```diff
@@ -14,48 +14,28 @@
 class RabbitMQEventManager(EventManager):
     def __init__(self, queue_uri: str):
         self.queue_uri = queue_uri
-        self.connection = self.connect()
+        self.connection = get_connection(self.queue_uri)
+        self.channel = self.connection.channel()
 
     def publish(self, event: Event) -> None:
         event_data = event.json()
         queue_name = self._queue_name(event)
 
         logger.debug("Publishing event: %s", event_data)
-
-        channel = self.connection.channel()
-        channel.queue_declare(queue_name)
+        self.channel.queue_declare(queue_name)
```

Comment:
Does the queue need to be re-declared for every publish?

Reply:
Not if those queue's are durable I'd guess, then we could just set them up once, or set (re_)declare them up on init.